src/Test/Integration/Model/Content/Generator/PostTest.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EasyTranslate\CompatMageplazaBlog\Test\Integration\Model\Content\Generator;
+
+use EasyTranslate\CompatMageplazaBlog\Model\Content\Generator\Posts as PostsGenerator;
+use EasyTranslate\CompatMageplazaBlog\Model\ResourceModel\Posts as CompatResourcePost;
+use EasyTranslate\Connector\Api\ProjectRepositoryInterface;
+use EasyTranslate\Connector\Model\Content\Generator\AbstractGenerator;
+use EasyTranslate\Connector\Model\Project;
+use Magento\TestFramework\Helper\Bootstrap;
+use Mageplaza\Blog\Api\Data\PostInterface;
+use Mageplaza\Blog\Model\Post;
+use Mageplaza\Blog\Model\PostFactory;
+use Mageplaza\Blog\Model\ResourceModel\Post as PostResource;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @magentoDbIsolation  enabled
+ */
+class PostTest extends TestCase
+{
+    /**
+     * @var int
+     */
+    private static $projectId;
+
+    /**
+     * @var ProjectRepositoryInterface
+     */
+    private $projectRepository;
+
+    /**
+     * @var PostsGenerator
+     */
+    private $postsGenerator;
+
+    /**
+     * @var PostFactory
+     */
+    private $postFactory;
+
+    /**
+     * @var PostResource
+     */
+    private $postResource;
+
+    /**
+     * @var CompatResourcePost
+     */
+    private $compatPostResource;
+
+    protected function setUp(): void
+    {
+        $objectManager            = Bootstrap::getObjectManager();
+        $this->projectRepository  = $objectManager->create(ProjectRepositoryInterface::class);
+        $this->postsGenerator     = $objectManager->create(PostsGenerator::class);
+        $this->postFactory        = $objectManager->create(PostFactory::class);
+        $this->postResource       = $objectManager->create(PostResource::class);
+        $this->compatPostResource = $objectManager->create(CompatResourcePost::class);
+    }
+
+    /**
+     * @magentoDataFixture    loadPostsFixture
+     * @magentoDataFixture    loadProjectFixture
+     * @magentoAppIsolation   enabled
```

Comment:
Why is that needed?

Reply:
We need this since there is some leftover data that is used in the other tests. This way the tests will not fail.